tasks/src/main/scala/dagr/tasks/ScatterGather.scala

```diff
@@ -122,24 +173,28 @@ object ScatterGather {
     * when the partitions are available and the set of Gathers to be performed on this stage of the Scatter.
     */
   private trait SubScatter[Result <: Task] extends Scatter[Result] {
+    // the tasks that create that results
```

Comment:
No - the tasks _are_ the results.

Reply:
That's a fact, good catch.  [Makes me think of this quote](https://en.wikipedia.org/wiki/The_medium_is_the_message)